components/ai_chat/core/browser/engine/engine_consumer_conversation_api_unittest.cc

```diff
@@ -189,7 +189,8 @@ TEST_F(EngineConsumerConversationAPIUnitTest, GenerateEvents_BasicMessage) {
   // ConversationEvent to JSON. It's convenient to test both here but more
   // exhaustive tests of  ConversationAPIClient are performed in its own
   // unit test suite.
-  std::string page_content(kTestingMaxAssociatedContentLength + 1, 'a');
+  PageContent page_content(
+      std::string(kTestingMaxAssociatedContentLength + 1, 'a'), false);
```

Comment:
we now need multiple page content event that sum of their size would exceed `kTestingMaxAssociatedContentLength`